packages/server/src/app.module.ts

```diff
@@ -11,6 +11,7 @@ import { AdminModule } from './admin/admin.module';
 import { CommandModule } from 'nestjs-command';
 import * as typeormConfig from '../ormconfig';
 import { AdminCommand } from 'admin/admin.command';
+import { ICalCommand } from 'course/ical.command';
```

Comment:
I might be missing something but it doesn't like this import, or the one above it is being used in this file.

Reply:
good catch